testing/execution.ts

```diff
@@ -312,20 +443,41 @@ async function executeFormulaOrSyncWithRawParamsInVM<
 
   const manifest = await importManifest(bundlePath);
   let params: ParamValues<ParamDefs>;
-  if (formulaSpecification.type === FormulaType.Standard) {
-    const formula = findFormula(manifest, formulaSpecification.formulaName);
-    params = coerceParams(formula, rawParams as any);
-  } else {
-    const syncFormula = findSyncFormula(manifest, formulaSpecification.formulaName);
-    params = coerceParams(syncFormula, rawParams as any);
+  let syncUpdates: GenericSyncUpdate[] | undefined;
+  switch (formulaSpecification.type) {
+    case FormulaType.Standard: {
+      const formula = findFormula(manifest, formulaSpecification.formulaName);
+      params = coerceParams(formula, rawParams as any);
+      break;
+    }
+    case FormulaType.Sync: {
+      const syncFormula = findSyncFormula(manifest, formulaSpecification.formulaName);
+      params = coerceParams(syncFormula, rawParams as any);
+      break;
+    }
+    case FormulaType.Metadata: {
+      // Interstingly we don't need special handling for the formula context dict (the optional second arg
```

Comment:
Nit/typo here and below: "Interstingly" -> "Interestingly"

Reply:
Oh yep thanks, fixing.